content/services/user-support.mdx

```diff
@@ -1,44 +1,93 @@
 ---
 title: User Support
 description: |
-  We strategically facilitate access to essential university research services.
+  We offer a variety of support options designed to fit your needs. Choose the method that works best for you.
 ---
 
-<ContentSection title="How We Support Brown">
+<ContentSection title="Documentation">
+  Our  <a href="https://docs.ccv.brown.edu/documentation/" target="_blank" rel="noopener noreferrer">documentation</a> 
+  is the fastest way to find answers to common questions about using our systems.
 
-We're here to help you navigate our services and make your research computing as seamless as possible. We provide
-support for our High-Performance Computing cluster, [Oscar](/services/oscar), our secure compute and storage system,
-[Stronghold](/services/stronghold), various [storage and transfer services](/services/storage) like Globus and
-Hibernate, and more!
+  <LinkList links={[
+  {label: "Stronghold", url: "https://docs.ccv.brown.edu/stronghold"},
+  {label: "Oscar", url: "https://docs.ccv.brown.edu/oscar"},
+  {label: "JupyterHub", url: "https://docs.ccv.brown.edu/jupyterhub"},
+  {label: "Globus", url: "https://docs.ccv.brown.edu/globus/"},
+  {label: "Hibernate", url: "https://docs.ccv.brown.edu/hibernate/"},
+  {label: "Starfish", url: "https://docs.ccv.brown.edu/starfish/"},
+  ]}/>
 
-We can assist with:
+</ContentSection>
 
-- **Consulting** on which of CCV systems best fits your research needs.
-- **Onboarding** to CCV systems.
-- **Troubleshooting** and debugging your issues and problems when using CCV systems.
+<ContentSection title="Direct Support">
 
-</ContentSection>
+  <CardGroup>
```

Comment:
I'm not sure about having these cards repeated in the middle of this page, since it is the same as the Contact Us page. Does it make sense to put it at the bottom or lower down so the other information is more prominent since it isn't anywhere else? Maybe this is fine whenever we have the sidebar menu though.